src/command_modules/azure-cli-redis/azure/cli/command_modules/redis/custom.py

```diff
@@ -37,3 +38,37 @@ def cli_redis_update_settings(client, resource_group_name, name, redis_configura
         existing.static_ip,
         )
     return client.create_or_update(resource_group_name, name, parameters=update_params)
+
+def cli_redis_create(client, resource_group_name, name, location, sku_name=None, # pylint:disable=too-many-arguments
+                     sku_family=None, sku_capacity=None, tags=None,
+                     redis_configuration=None, enable_non_ssl_port=None, tenant_settings=None,
+                     shard_count=None, subnet_id=None, static_ip=None):
+    # pylint:disable=line-too-long
+    """Create new Redis Cache instance
+    :param resource_group_name: Name of resource group
+    :param name: Name of redis cache
+    :param location: Location
+    :param sku_name: What type of redis cache to deploy. Valid values: (Basic, Standard, Premium).
+    :param sku_family: Which family to use. Valid values: (C, P).
+    :param sku_capacity: What size of redis cache to deploy. Valid values for C family (0, 1, 2, 3, 4, 5, 6), for P family (1, 2, 3, 4)
+    :param redis_configuration: All Redis Settings. Few possible keys rdb-backup-enabled,rdb-storage-connection-string,rdb-backup-frequency,maxmemory-delta,maxmemory-policy,notify-keyspace-events,maxmemory-samples,slowlog-log-slower-than,slowlog-max-len,list-max-ziplist-entries,list-max-ziplist-value,hash-max-ziplist-entries,hash-max-ziplist-value,set-max-intset-entries,zset-max-ziplist-entries,zset-max-ziplist-value etc.
```

Comment:
>Few possible keys rdb-backup-enabled,rdb-storage-connection-string,rdb-backup-frequency,maxmemory-delta,maxmemory-policy,notify-keyspace-events,maxmemory-samples,slowlog-log-slower-than,slowlog-max-len,list-max-ziplist-entries,list-max-ziplist-value,hash-max-ziplist-entries,hash-max-ziplist-value,set-max-intset-entries,zset-max-ziplist-entries,zset-max-ziplist-value etc. [](start = 52, length = 373)

These wrap strangely when you specify -h, I'm assuming because there aren't spaces after the commas.